1-estrategia/README.md

```diff
@@ -2,22 +2,129 @@
+
+
 ### Circunstancias de uso
 
+__Quien__:
+
+El público al que nos dirigimos son migrantes de entre 20 y 45 años que llegan al País Vasco. Nos hemos centrado en esta franja de edad, ya que según una encuesta realizada por el Gobierno Vasco en el año 2010 el 47% de los inmigrantes que llegan tienen entre 25 y 44 años, un 34% son menores de 25 años y tan solo el 19% son mayores de 45 años. Por esta razón, hemos concretado nuestro publico objetivo en este rango de edad, ya que además son las personas que más uso hacen de plataformas web, redes sociales y aplicaciones para móvil o tablets.
+
+__Que__: 
+
+La plataforma web de Aupa , informará a las personas migrantes que se encuentran en el País Vasco. Es una plataforma creada con el fin de que las personas afectadas encuentren ayudas administrativas, refugio, educación, alimentación, asistencia sanitaria o que puedan incorporarse a organizaciones que defiendan sus derechos, como ACNUR u otras ONGs.
+
+__Cuando__: 
+
+Desde el momento en que llegan al País Vasco para instalarse, debido a que la plataforma se centra en ayudar a acoplarse en la nueva cultura y también en la reubicación para que así puedan satisfacer sus necesidades básicas. 
+
+__Dónde__: 
+
+Pais Vasco. 
+
+__Para que__: 
+
+Facilitar la integración de los inmigrantes en la comunidad Vasca, brindarles esperanza y que puedan superar la crisis en la que se ven sumergidos.
+
+
 ### Criterios de diseño
 
+Aupa tiene como fin comunicar e informar a los migrantes del País Vasco, por ende los criterios de diseño los definimos de la siguiente manera: 
+
+* Accesibilidad: El sitio web está pensado para obtener acceso multipantalla, es decir, nuestros usuarios pueden ingresar al sitio web desde la computadora, la tablet o el teléfono móvil. 
+
+* Findability:  En nuestro sitio web, la intuitividad es muy importante, las pestañas deben ser muy claras y con una organización fácil para los inmigrantes. 
+
+* Funcionalidad: se refiere a que todo lo disponible en la web funcione correctamente. 
+
+* Utilidad: hace referencia al uso que el usuario pueda dar a la web. Es decir, que la información y la manera de ordenarla sea correcta para cumplir con las necesidades del público objetivo.
+
+* Estética: Esta cualidad busca una apariencia amigable para el usuario. La elección de la tipografía es un ejemplo de criterio influenciado por este factor. → Para nuestro sitio web, la simplicidad es un factor muy importante porque no todos nuestros usuarios dominan la lengua hispana, por esta razón no integraremos textos largos, a su vez, complementaremos con imágenes y multimedia para enriquecer nuestro sitio y ayudar a que se pueda encontrar de una manera más rápida lo que buscan. Es decir, el diseño de la plataforma está pensado en simplicidad desde las pestañas del menú hasta el contenido informativo.
```

Comment:
Lo que comentáis poco tiene que ver con la estética. Buscad una manera más apropiada de denominar este criterio de diseño.